tailwind.config.js

```diff
@@ -0,0 +1,36 @@
+const defaultTheme = require('tailwindcss/defaultTheme');
+const colors = require('tailwindcss/colors');
+
+module.exports = {
+  purge: [
+    './index.html',
+    './src/**/*.{vue,js,ts,jsx,tsx}',
+    './node_modules/litepie-datepicker/**/*.js',
```

Comment:
You don't have the litepie-datepicker in your project (yet)

Reply:
my bad i will remove it :sweat_smile: